packages/discovery-provider/plugins/pedalboard/apps/trending-challenge-rewards/package.json

```diff
@@ -28,6 +28,7 @@
     "knex": "^2.4.2",
     "moment": "^2.29.4",
     "morgan": "^1.10.0",
+    "node-cron": "^3.0.3",
     "ts-results": "^3.3.0"
```

Comment:
lot of these are probably me but can we pin?

Reply:
sure thing

Reply:
I tried pinning and... some stuff broke. I think probably a later problem...